lib/galaxy/webapps/tool_shed/app.py

```diff
@@ -64,6 +64,7 @@ def __init__(self, **kwd):
         from galaxy.managers.citations import CitationsManager
         self.citations_manager = CitationsManager(self)
         # The Tool Shed makes no use of a Galaxy toolbox, but this attribute is still required.
+        self.config.use_tool_dependencies = False
```

Comment:
This is OK for this edit. However, in the long run, I think we should try to keep any modifications to the app.config object to the modules that load it at startup (i.e., files `config/__init__.py` for galaxy and `config.py` for TS and reports).

Reply:
In the original suggestion from @jmchilton , this was `self.app.use_tool_dependencies`, I can just back to that if you prefer, no strong opinion from me.

Reply:
No preference. Now that I think of it, it could belong in either one, and config seems reasonable, I think. (I think if it were to go into the app object, that would imply that it is not a configurable property in the first place - and that's a discussion that, I think, belongs outside of this PR)

Reply:
I've restored John's version, I think it is suppose to mean that the Tool Shed app does not support tool dependency resolution, so it's not configurable. On Galaxy instead, it is indeed supported and configurable.

Reply:
OK, that makes sense.